docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -163,6 +163,20 @@ Create the role:
 $ tctl create -f auto-users.yaml
 # role 'auto-users' has been created
 ```
+### Sudoers Configuration
```

Comment:
This doesn't seem like the right place for this information. This H2 section is about configuring RBAC, and this H3 is not—it appears unexpectedly between the H3s titled `Define a Teleport role` and `Create a Teleport user`. 

I would suggest adding a new Step 1 called "Prepare your system" (or similar) that instructs the user to ensure that their Linux servers can accommodate host user creation and includes this information.